runtime/cudaq/builder/kernel_builder.h

```diff
@@ -444,24 +453,27 @@ class kernel_builder : public details::kernel_builder_base {
     QuakeValue v(*opBuilder, param);                                           \
     details::NAME(*opBuilder, v, empty, qubit);                                \
   }                                                                            \
-  template <NumericType ConstantParameter>                                     \
-  void NAME(ConstantParameter &&constant, QuakeValue qubit) {                  \
-    NAME(constant, qubit);                                                     \
```

Comment:
This overload caused an infinite recursive warning when the constant is a `float`, for example. The `void NAME(double param, QuakeValue qubit)` overload seems to work for all floating point types.